blocks/logic.js

```diff
@@ -480,7 +480,7 @@ Blockly.Constants.Logic.CONTROLS_IF_EXTENSION = function() {
   }.bind(this));
 };
 
-Blockly.Extensions.register('controls_if',
+Blockly.Extensions.register('controls_if_mutator',
   Blockly.Constants.Logic.CONTROLS_IF_EXTENSION);
```

Comment:
Renamed.

My thought process: the extension is more than just the mutator; it is the tooltip, also.  But I had the same thought/hesitation when I made this edit. Happy to make the change.